### PR TITLE
[DSD-6004] Updated browserstack-web.yml to push artifacts to s3 bucket.

### PR DESCRIPTION
**Pull request by Mohanraj209**

### CODE REVIEW

File: .github/workflows/browserstack-web.yml [hunk 1]
```diff
@@ -37,4 +42,15 @@
...
-          path: inji-web-test/test-output          
+          path: inji-web-test/test-output
+          
+      - name: Upload to S3
+        if: success() || failure()
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          AWS_REGION: "ap-south-1" 
+          S3_BUCKET: "inji-stack"
```
Comment (syedsalman3753) [inline .github/workflows/browserstack-web.yml:53]: @Mohanraj209 

please take the above variables as github secrets

File: .github/workflows/browserstack-web.yml [hunk 2]
```diff
@@ -37,4 +42,15 @@
...
+          path: inji-web-test/test-output
+          
+      - name: Upload to S3
+        if: success() || failure()
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          AWS_REGION: "ap-south-1" 
+          S3_BUCKET: "inji-stack" 
+          S3_OBJECT_KEY: "inji-web"
```
Comment (syedsalman3753) [inline .github/workflows/browserstack-web.yml:54]: please take the above variable as a repo name

**Review by syedsalman3753 — CHANGES_REQUESTED** [review]

File: .github/workflows/browserstack-web.yml (hunk 1 above)
Comment (Mohanraj209) [reply .github/workflows/browserstack-web.yml]: updated

File: .github/workflows/browserstack-web.yml (hunk 2 above)
Comment (Mohanraj209) [reply .github/workflows/browserstack-web.yml]: updated

**Review by syedsalman3753 — APPROVED** [review]

**Review by ckm007 — APPROVED** [review]

**Pull request merged**